src/main/kotlin/fi/metatavu/timebank/api/persistence/model/VacationRequest.kt

```diff
@@ -0,0 +1,52 @@
+package fi.metatavu.timebank.api.persistence.model
+
+import fi.metatavu.timebank.model.RequestStatus
+import fi.metatavu.timebank.model.VacationType
+import java.time.LocalDate
+import java.time.OffsetDateTime
+import java.util.*
+import javax.persistence.*
+
+/**
+ * VacationRequest JPA entity
+ */
+@Entity
+class VacationRequest (
```

Comment:
Use lateinit modifier on all non-nullable non-primitive types to avoid all of the null assertions later.

Reply:
lateinit modifier is not allowed on primary constructor parameters. They also require a default value.